models/classes/accessControl/Service/AccessTokenService.php

```
@@ -0,0 +1,83 @@
+        return $content;
+    }
+
+    public function extractAccessTokenFromRequest(): string
+    {
+        $authorizationHeader = explode(' ', $_SERVER['HTTP_AUTHORIZATION'] ?? '', 2);
+        return array_pop($authorizationHeader);
+    }
+
+    public function extractAccessTokenPayloadFromRequest(): array
+    {
+        @[$_, $payload] = explode('.', $this->extractAccessTokenFromRequest());
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_'. (undefined)

(UnusedLocalVariable)